src/Broadway/EventStore/IlluminateEventStore.php

```diff
@@ -0,0 +1,119 @@
+<?php namespace App\Broadway\EventStore;
+
+use Broadway\Domain\DomainEventStream;
+use Broadway\Domain\DomainEventStreamInterface;
+use Broadway\Domain\DomainMessage;
+use Broadway\Domain\DateTime;
+use Broadway\EventStore\EventStoreInterface;
+use Broadway\EventStore\EventStreamNotFoundException;
+use Broadway\Serializer\SerializerInterface;
+use Illuminate\Config\Repository;
+use Illuminate\Database\DatabaseManager;
+
+/**
+ * Class IlluminateEventStore
+ *
+ * Create a broadway.php config file in your config directory to change the table name
+ *
+ * @author  Dennis Schepers
+ * @package App\Broadway\EventStore
+ */
+class IlluminateEventStore implements EventStoreInterface {
+
+    /**
+     * @var DatabaseManager
+     */
+    protected $databaseManager;
+
+    /**
+     * @var \Broadway\Serializer\SerializerInterface
+     */
+    protected $serializer;
+
+    /**
+     * @var \Illuminate\Contracts\Config\Repository
+     */
+    protected $config;
+
+    /**
+     * Construct the dependancies
+     *
+     * @param \Illuminate\Database\DatabaseManager $databaseManager
+     * @param \Illuminate\Config\Repository $configRepository
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(
+        DatabaseManager $databaseManager,
+        Repository $configRepository,
+        SerializerInterface $serializer
+    ) {
+        $this->databaseManager = $databaseManager;
+        $this->config = $configRepository;
+        $this->table = $this->config->get('broadway.table','event');
```

Comment:
I don't know enough about Laravel and friends, but wouldn't it be possible to inject:
- the database connection
- the table

Instead of the database manager and the configuration?